plugin-server/src/types.ts

```diff
@@ -487,42 +512,38 @@ export interface EventMessage extends BaseEventMessage {
     sent_at: DateTime | null
 }
 
-/** Raw Organization row from database. */
-export interface RawOrganization {
-    id: string
-    name: string
-    created_at: string
-    updated_at: string
-    available_features: string[]
-}
-
-/** Usable Team model. */
-export interface Team {
-    id: number
+/** Properties shared by RawEvent and Event. */
+interface BaseEvent {
     uuid: string
-    organization_id: string
-    name: string
-    anonymize_ips: boolean
-    api_token: string
-    slack_incoming_webhook: string
-    session_recording_opt_in: boolean
-    ingested_event: boolean
+    event: string
+    team_id: number
+    distinct_id: string
+    /** Person UUID. */
+    person_id?: string
 }
 
-/** Re-export Element from scaffolding, for backwards compat. */
-export { Element } from '@posthog/plugin-scaffold'
-
-/** Usable Event model. */
-export interface Event {
-    id: number
-    event?: string
-    properties: Record<string, any>
-    elements?: Element[]
+/** Raw event row from ClickHouse. */
+export interface RawEvent extends BaseEvent {
     timestamp: string
-    team_id: number
-    distinct_id: string
-    elements_hash: string
     created_at: string
+    properties?: string
+    elements_chain: string
+    person_created_at?: string
+    person_properties?: string
+    group0_properties?: string
+    group1_properties?: string
+    group2_properties?: string
+    group3_properties?: string
+    group4_properties?: string
+}
+
+/** Parsed event row from ClickHouse. */
+export interface Event extends BaseEvent {
```

Comment:
Probably should be `ParsedClickhouseEvent` with the other type being RawClickhouseEvent. As-is it just is confusing when seeing it in code.